### ISSUE TITLE
Links on the "Nodes" table on "Overview" don't go to the correct place

### ISSUE THREAD

**Issue by ziggythehamster** [scylladb/scylla-monitoring#1122]
In Scylla Monitoring 3.5's Overview dashboard, the Nodes table has handy links. Unfortunately, they don't work, at least with 2020.1.

The CQL link is:

```
http://foo:3000/d/cql-2020.1/scylla-cql?refresh=30s&orgId=1&var-by=instance&var-node=10.20.7.16
```

This should be:

```
http://foo:3000/d/cql-2020-1/scylla-cql?refresh=30s&orgId=1&var-by=instance&var-node=10.20.7.16
```

The node detail link is:

```
http://foo:3000/d/detailed-2020.1/Detailed?refresh=30s&orgId=1&var-by=instance&var-node=10.20.7.16
```

It should be:

```
http://foo:3000/d/detailed-2020-1/detailed?refresh=30s&orgId=1&var-by=instance&var-node=10.20.7.16
```

And finally, the version link is:

```
http://foo:3000/d/OS-2020.1/OS-metrics?refresh=30s&orgId=1&var-by=instance&var-node=10.20.7.16
```

It should be:

```
http://foo:3000/d/OS-2020-1/os-metrics?refresh=30s&orgId=1&var-by=instance&var-node=10.20.7.16
```

**Comment by ziggythehamster** [comment]
This is not fixed by #1124.

**Comment by amnonh** [comment]
I've just tested it again with 3.5 with the latest changes and it seems to work correctly, please make sure you are using the right dashboard and that you refresh your browser.
